test/utils/computeLedgerHash/index.ts

```diff
@@ -1,35 +1,35 @@
 import assert from 'assert-diff'
+import {computeLedgerHeaderHash} from '../../../src'
+import {ValidationError} from '../../../src/common/errors'
 import requests from '../../fixtures/requests'
 import responses from '../../fixtures/responses'
-import {assertResultMatch, TestSuite} from '../../utils'
+import getLedgerFull from '../../fixtures/responses/get-ledger-full.json'
+import {assertResultMatch} from '../../utils'
 const {computeLedgerHash: REQUEST_FIXTURES} = requests
 
 function getNewLedger() {
-  return JSON.parse(JSON.stringify(responses.getLedger.full))
+  return JSON.parse(JSON.stringify(getLedgerFull))
```

Comment:
These two lines are identical, there's no need to change it - it's a little easier to understand with the old version, actually